src/components/newsItem.js

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import { Link } from "gatsby"
+import "./newsItem.scss"
+
+export default function NewsItem({
+  image,
+  date,
+  title,
+  description,
+  id,
+  hoverEffects,
+}) {
+  return (
+    <div className={hoverEffects ? "news-item news-item--hover" : "news-item"}>
```

Comment:
We need to think about this for accessibility. The hidden link arrow on desktop is a big problem for keyboard users. I think we should do two things:

1. Make the enter item a link
2. Make sure the hover effect occurs when tabbing onto an item as well.